index.html

```diff
@@ -36,5 +36,151 @@ <h5 class="connect">LET'S CONNECT</h5>
+          <ul class="info">
+              <li class="tonic-detail">CANOPY</li>
+              <li><img src="./img/Counter.png" alt="punto"></li>
+              <li class="tonic-detail2">Back End Dev</li>
+              <li><img src="./img/Counter.png" alt="counter"></li>
+              <li class="tonic-detail2">2015</li>
+          </ul>
+          <p class="project-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div>
+              <ul class="buttons">
+                  <li><button class="int-button">html</button></li>
+                  <li><button class="int-button">css</button></li>
+                  <li><button class="int-button">JavaScript</button></li>
+              </ul>
+          </div>
+          <div><button class="see">See Project</button>
+          </div>
+       </div>
+
+
+          <div class="water2">
+            <div class="project2"><img src="./img/Snapshoot-project2.png" alt="project2"></div>
+            <h2 class="multi-post">Multi-Post Stories</h2>
+            <ul class="info">
+              <li class="tonic-detail">CANOPY</li>
+              <li><img src="./img/Counter.png" alt="punto"></li>
+              <li class="tonic-detail2">Back End Dev</li>
+              <li><img src="./img/Counter.png" alt="counter"></li>
+              <li class="tonic-detail2">2015</li>
+            </ul>
+            <p class="project-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div>
+                <ul class="buttons">
+                    <li><button class="int-button">html</button></li>
+                    <li><button class="int-button">css</button></li>
+                    <li><button class="int-button">JavaScript</button></li>
+                </ul>
+            </div>
+            <div><button class="see">See Project</button>
+            </div>
+          </div>
+
+          <div class="water3">
+            <div class="project3"><img src="./img/Snapshoot-project3.png" alt="project3"></div>
+            <h2 class="tonic">Tonic</h2>
+            <ul class="info">
+              <li class="tonic-detail">CANOPY</li>
+              <li><img src="./img/Counter.png" alt="punto"></li>
+              <li class="tonic-detail2">Back End Dev</li>
+              <li><img src="./img/Counter.png" alt="counter"></li>
+              <li class="tonic-detail2">2015</li>
+            </ul>
+            <p class="project-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div>
+                <ul class="buttons">
+                    <li><button class="int-button">html</button></li>
+                    <li><button class="int-button">css</button></li>
+                    <li><button class="int-button">JavaScript</button></li>
+                </ul>
+            </div>
+            <div><button class="see">See Project</button>
```

Comment:
- [ ] Please try to stick to the design as much as possible to make your work professional, try to fix this button so that it appears in the project card, not on side.

your design:
![image](https://user-images.githubusercontent.com/90524466/185734409-e2c2f579-aecf-4e75-ab76-8eeef9b424d8.png)

Figma design:
![image](https://user-images.githubusercontent.com/90524466/185734458-6c472f93-0134-4c99-badf-aa0bf74d31de.png)